File: C/PRAK405-2210817310017-Rohid_Akmal.c
#include <stdio.h>

void main(){
    int a, b, i, j, o, p, q, hsl, t;

    scanf("%d %d", &a, &b);
    for(i = 1; i <= a; i++){
        for(j = i; j > 1; j--){printf("(%d * %d) +", j, b);}
        for(o = 1, hsl = o * b; o < i; o++, hsl += (o*b));{
        printf("(%d * %d) = %d\n", j, b , hsl);}
    for(p = 1, q = 1, t = 0; p <= a; q += p + 1, p++){
        t += q * b;}}
    printf("%d", t);
}

File: Python/PRAK405-2210817310017-Rohid Akmal.py
a,b = input().split()
a = int(a) ; b = int(b)
for i in range(1, a + 1) :
    j = i
    while j > 1 :
        print("(%d * %d) + "%(j, b), end='') ; j -= 1
    o = 1 ; hsl = (i * b)
    while o < i :
        hsl += (o * b) ; o += 1
    print("(%d * %d) = %d"%(j, b, hsl))
p = 1 ; q = 1; t = 0
while p <= a :
    t += q * b ; q += p + 1 ; p +=1
print("%d"%(t))